lasagne/layers/shape.py

```diff
@@ -21,17 +21,33 @@
 
 class FlattenLayer(Layer):
     """
-    Flatten all but the first dimension.
+    Flatten all dimensions trailing ``dim``
```

Comment:
Yes, If dim is 0 all dimensions are flattened and 1 is the original behavior.

I'll try to come of with a more precise description.